<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="imagesListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ImageListLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="trainButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="recognizeImageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="recognizedImageLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To recognize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognized:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trained images:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFigures">
    <property name="title">
     <string>Figures</string>
    </property>
    <addaction name="actionAdd_figure"/>
    <addaction name="actionRecognize_figure"/>
   </widget>
   <addaction name="menuFigures"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_figure">
   <property name="text">
    <string>Add figure</string>
   </property>
  </action>
  <action name="actionRecognize_figure">
   <property name="text">
    <string>Recognize figure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
